programs/bpf_loader/src/lib.rs

```diff
@@ -1498,6 +1364,189 @@ fn process_loader_upgradeable_instruction(
     Ok(())
 }
 
+fn common_extend_program(
+    invoke_context: &mut InvokeContext,
+    additional_bytes: u32,
+    check_authority: bool,
+) -> Result<(), InstructionError> {
+    let log_collector = invoke_context.get_log_collector();
+    let transaction_context = &invoke_context.transaction_context;
+    let instruction_context = transaction_context.get_current_instruction_context()?;
+    let program_id = instruction_context.get_last_program_key(transaction_context)?;
+
+    const PROGRAM_DATA_ACCOUNT_INDEX: IndexOfAccount = 0;
+    const PROGRAM_ACCOUNT_INDEX: IndexOfAccount = 1;
+    const AUTHORITY_ACCOUNT_INDEX: IndexOfAccount = 2;
+    // let system_program_account_index = if check_authority { 3 } else { 2 };
```

Comment:
ought not be committing commented code

Reply:
This was an explicitly requested hint: https://github.com/anza-xyz/agave/pull/6131#discussion_r2084369421

Reply:
ok, but why is it in rust instead of English tho?

Reply:
Rust is easier to understand than plain English, obviously xD

Reply:
Well I can change the comment if you insist. That would have to start in master again and then be backported too.